ledger/ledger-runner-common/src/main/scala/com/daml/ledger/runner/common/CliConfig.scala

```diff
@@ -56,6 +56,8 @@ final case class CliConfig[Extra](
     timeProviderType: TimeProviderType,
     tlsConfig: Option[TlsConfiguration],
     userManagementConfig: UserManagementConfig,
+    // TODO LLP: Move to `ParticipantConfig`
+    maxTransactionsInMemoryFanOutBufferSize: Int,
```

Comment:
`CliParticipantConfig` then, as `ParticipantConfig` wont have it - it would stay in `IndexServiceConfig` for that level